plugins/main/bridge/bridge_test.go

```diff
@@ -1672,6 +1764,25 @@ var _ = Describe("bridge Operations", func() {
 		Expect(testutils.UnmountNS(targetNS)).To(Succeed())
 	})
 
+	var (
+		correctMinID int = 100
+		correctMaxID int = 105
+	)
+
+	DescribeTable(
+		"collectVlanTrunk succeeds",
+		func(vlanTrunks []*VlanTrunk, expectedVIDs []int) {
+			Expect(collectVlanTrunk(vlanTrunks)).To(Equal(expectedVIDs))
+		},
+		Entry("", &[]*VlanTrunk{}, []int{}),
+		Entry("", &[]*VlanTrunk{
```

Comment:
please fill the entry descriptions; otherwise, if the tests fail, we will not know which of them has failed.

Reply:
I'm trial and error right now.
I'm not sure the reason I cannot test it on my PC.
So I'm asking github action to help.
Please hold on.

Reply:
Sorry to bother.
I used another branch to test right now.

Reply:
I used my repo to run action.
so it works to test.